### PR TITLE
Backup Stack

### PR DESCRIPTION
**Pull request by caiovfernandes**
- Add aws backup stack.

### CODE REVIEW

File: interfaces/lib/aws_backup/interfaces.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export interface BackupStackConfig {
+  stackName: string;
+  backups: BackupConfig[];
+}
+
+export interface BackupConfig {
+  name: string;
+  rule_schedule: string;
```
Comment (harshadbhatia) [inline interfaces/lib/aws_backup/interfaces.ts:8]: can we rename this to ruleSchedule

File: interfaces/lib/aws_backup/interfaces.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export interface BackupStackConfig {
```
Comment (harshadbhatia) [inline interfaces/lib/aws_backup/interfaces.ts:1]: This should be BackupStackProps

File: interfaces/lib/aws_backup/interfaces.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  stackName: string;
+  backups: BackupConfig[];
+}
+
+export interface BackupConfig {
```
Comment (harshadbhatia) [inline interfaces/lib/aws_backup/interfaces.ts:6]: Same as above. BackupProps

**Review by harshadbhatia — CHANGES_REQUESTED** [review]

**Comment by caiovfernandes** [comment]
Hey @harshadbhatia, I'm closing this PR, as it was to be opened at https://github.com/keshav-1835i/improved-cdk-constructs.

I've opened this one now (https://github.com/keshav-1835i/improved-cdk-constructs/pull/3) with your requested changes, feel free to review it too, thanks!

**Pull request closed without merging**